src/log/log.go

```diff
@@ -0,0 +1,34 @@
+package logger
+
+import (
+	"fmt"
+	"log"
+)
+
+const (
+	// ANSI color codes
+	Red    = "\033[31m"
+	Green  = "\033[32m"
+	Yellow = "\033[33m"
+	White  = "\033[97m"
+	Reset  = "\033[0m"
+)
+
+func Printf(color, format string, v ...interface{}) {
+	msg := fmt.Sprintf(format, v...)
+	log.Printf(color + msg + Reset)
+}
+
+func Println(color string, args ...interface{}) {
+	var msg string
+	for _, v := range args {
+		str, _ := v.(string)
+		msg += str
+	}
+	log.Println(color + msg + Reset)
```

Comment:
```suggestion
	msg := color + fmt.Sprint(args...) + Reset
	log.Println(msg)
```
or simply: `log.Println(color + fmt.Sprint(args...) + Reset)`

Reply:
Not sure if I understand your point. There is no formatter needed. Are you confusing `fmt.Sprint` with `fmt.Sprintf`?

Reply:
You're right! It was my confusion, sorry! hehe